mteb/cmd.py

```diff
@@ -20,6 +23,25 @@
 logger = logging.getLogger(__name__)
 
 
+def _name_to_path(name: str) -> str:
+    return name.replace("/", "__").replace(" ", "_")
```

Comment:
I think you can remove `sentence_transformers` prefix from the model name. Just as in `mteb/results` (we already have the info in the metadata)

Reply:
okay 👍